### DISCUSSION TITLE
Welcome to whippr Discussions!

### DISCUSSION THREAD

**Discussion by fmmattioni** [fmmattioni/whippr#5] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Chamoux74** [comment]
Hi,

Thank you very much for developing this package.

I haven't come across this information—perhaps it isn't included...
I would like to know whether outliers are removed or replaced with a value interpolated based on the moving average (or using another method) in order to preserve the time series ?

Many thanks
